dts/riscv/openhwgroup/cva6.dtsi

```diff
@@ -0,0 +1,152 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2024 CISPA Helmholtz Center for Information Security
+ */
+
+/*
+ * CVA6 SoC without CPU, which comes in 32 and 64 bit variants
+ */
+
+/ {
+	#address-cells = <0x01>;
+	#size-cells = <0x01>;
+
+	clocks {
+
+		clk_cpu {
+			#clock-cells = <0x00>;
+			clock-frequency = <50000000>;
+			compatible = "fixed-clock";
+		};
+
+		clk_bus: clk_bus_0 {
+			#clock-cells = <0x00>;
+			clock-frequency = <50000000>;
+			compatible = "fixed-clock";
+		};
+	};
+
+	soc {
+		#address-cells = <0x01>;
+		#size-cells = <0x01>;
+		ranges;
+
+		/* Xilinx MIG memory controller */
+		memory0: memory@80000000 {
+			device_type = "memory";
+			reg = <0x80000000 0x40000000>;
+			status = "okay";
+		};
+
+		/* RISC-V Platform-level interrupt controller */
+		plic: interrupt-controller@c000000 {
+			compatible = "sifive,plic-1.0.0";
+			#address-cells = <0>;
+			#interrupt-cells = <2>;
+			interrupt-controller;
+			interrupts-extended = <&hlic 11 &hlic 9>;
+			reg = <0x0c000000 0x4000000>;
+			riscv,max-priority = <7>;
+			riscv,ndev = <30>;
+			status = "okay";
+		};
+
+		/* USB UART */
+		uart0:serial@10000000 {
+			clock-frequency = <50000000>;
+			clocks = <&clk_bus>;
+			compatible = "ns16550";
+			current-speed = <115200>;
+			device_type = "serial";
+			reg = <0x10000000 0x10000>;
+			reg-shift = <0x02>;
+			parity = "none";
+			stop-bits = "1";
+			data-bits = <8>;
+			interrupt-parent = <&plic>;
+			interrupts = <1 0x04>;
+			status="okay";
```

Comment:
disabled, devices should be disabled in dts folder files then enabled in board dts unless they are required to operate e.g. clock

Reply:
Done